src/mcts/params.cc

```diff
@@ -91,9 +91,14 @@ const OptionId SearchParams::kTemperatureId{
     "while making the move."};
 const OptionId SearchParams::kTempDecayMovesId{
     "tempdecay-moves", "TempDecayMoves",
-    "Reduce temperature for every move from the game start to this number of "
-    "moves, decreasing linearly from initial temperature to 0. A value of 0 "
-    "disables tempdecay."};
+    "Reduce temperature for every move from decay start move, decreasing "
+    "linearly over this number of moves from initial temperature to 0. "
+    "A value of 0 disables tempdecay."};
+const OptionId SearchParams::kTempDecayStartMoveId{
+    "tempdecay-start-move", "TempDecayStartMove",
+    "Start the linear decrease of temperature after this number of moves, "
```

Comment:
It seems wrong to number moves from 0?
Also to me "starting decay from move 2" sounds like move should already receive reduced temperature at move, and I guess that doesn't happen here.


E.g. currently if --temp-decay-start-move=1, then the first move to recieve reduced temperature is actually move 3.

Reply:
Maybe rename it to temp decay delay? - 0 means the decay line attaches at move 1, 1 means it attaches at move 2 (3rd ply)

Reply:
I would be fine with either, renaming the variable or correct the off-by-one error explicitly.

> E.g. currently if --temp-decay-start-move=1, then the first move to recieve reduced temperature is actually move 3.

but this already means that we have an off by one error already in the current code, right?